src/table/src/requests.rs

```diff
@@ -106,3 +106,12 @@ pub struct DeleteRequest {
     /// The key is the column name, and the value is the column value.
     pub key_column_values: HashMap<String, VectorRef>,
 }
+
+/// Copy table request
+#[derive(Debug)]
+pub struct CopyTableRequest {
+    pub catalog_name: String,
+    pub schema_name: String,
+    pub table_name: String,
+    pub file_name: String,
```

Comment:
Looks like it should be the file path rather than file name?

Reply:
I think it's still a filename, or the parent name of some list of files. If we change it to a path_name, then it still needs a variable to represent the parent name of the files.